t/24-worker-engine.t

```diff
@@ -103,5 +107,11 @@ subtest 'asset settings' => sub {
     is_deeply($got, $expected, 'Asset settings are correct (no UEFI or NUMDISKS)') or diag explain $got;
 };
 
+subtest 'caching' => sub {
+    is(OpenQA::Worker::Engines::isotovideo::cache_assets, undef, 'cache_assets has nothing to do without assets');
+    my %assets = (ISO => 'foo.iso',);
+    my $got    = OpenQA::Worker::Engines::isotovideo::cache_assets(undef, undef, \%assets, undef, undef);
```

Comment:
I have a question about the second undef. The $vars is undef when doing $vars->{}. Is it better to pass a defined values to $vars?

Reply:
I guess it's ok this way for testing.